src/functions-and-arrays.js

```diff
@@ -78,7 +145,23 @@ const wordsCount = [
   'matter'
 ];
 
-function howManyTimes() {}
+function howManyTimes(array) {
+if (array.length === 0) {
+  return 0;
+}
+let count = 0;
+
+for (let i = 0; i < array.length; i++) {
+  for (let j = 0; j < array.length; j++) {
```

Comment:
and with the other parameter you won't need the second loop

```suggestion
  if (array[i] === search) {
    count ++;
  } 
```